save-frontend/src/main/kotlin/com/saveourtool/save/frontend/routing/MobileRouting.kt

```diff
@@ -0,0 +1,31 @@
+/**
+ * All routs for the mobile version of the frontend
+ */
+
+package com.saveourtool.save.frontend.routing
+
+import com.saveourtool.save.frontend.components.mobile.AboutUsMobileView
+import com.saveourtool.save.frontend.components.mobile.WelcomeMobileView
+import com.saveourtool.save.validation.FrontendRoutes
+import react.VFC
+import react.create
+import react.react
+import react.router.Route
+import react.router.Routes
+
+/**
+ * Just put a map: View -> Route URL to this list
+ */
+val mobileRoutes = VFC {
+    Routes {
+        listOf(
+            AboutUsMobileView::class.react.create() to FrontendRoutes.ABOUT_US.path,
```

Comment:
@petertrr @nulls @sanyavertolet  if you will have any ideas of how to cast it, so I won't need to have `.react.create()`, will be great. Simple map does not work, due to receiver type...

```
.map { it.react.create() }
```

Reply:
migrate to `FC`

Reply:
Maybe wrap it into `VFC`s? In that way it seems possible to easily iterate over the list and create components

Reply:
> migrate to `FC`

It seems to be a long journey, can we do it now?

Reply:
> Maybe wrap it into `VFC`s? In that way it seems possible to easily iterate over the list and create components

wrapper looks much more ugly, I want to remove existing actually

Reply:
> Simple map does not work, due to receiver type...

What type is inferred in this case? `.react` is an extension on `KClass<out Component<P, *>>`, maybe list can be cast somehow?